src/Compilers/CSharp/Portable/BoundTree/DecisionTreeBuilder.cs

```diff
@@ -165,15 +159,16 @@ private DecisionTree AddByValue(DecisionTree.ByType byType, BoundConstantPattern
                 {
                     if (byType.Default.MatchIsComplete)
                     {
+                        // This code may be unreachable due to https://github.com/dotnet/roslyn/issues/16878
```

Comment:
If this is not covered by fixing that issue, this code or comment would be changed as needed.